src/components/InjectedComponents/DecryptedPost.tsx

```diff
@@ -161,6 +164,10 @@ export function DecryptPost(props: DecryptPostProps) {
         undefined,
     )
 
+    const [postPayload, setPostPayload] = useState(deconstructPayload(encryptedText, null))
```

Comment:
if like this, `deconstructPayload(encryptedText, null)` will still have to be called every re-render (as it is an expression, its value needs to be evaluated before becomes an argument)... should be `() =>deconstructPayload(encryptedText, null)` so that react will detect the need of evaluating it.